src/composite.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SRC_COMPOSITE_H_
+#define SRC_COMPOSITE_H_
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/*
+  Composite images `src` and `dst`.
+*/
+int Composite(VipsObject *context, VipsImage *src, VipsImage *dst, VipsImage **out);
```

Comment:
Getting used to your 140 line length limit. I usually operate at 80 out of habit :wink: